### ISSUE TITLE
As of 12/9/19 - No longer working

### ISSUE THREAD

**Issue by TheBigShort111** [lawfx/YoutubeNonStop#20]
Seems to have stopped working as of 12/9 - Was working perfectly up till then. Firefox update have anything to do with it? Or perhaps YouTube changed something?

**Comment by TheBigShort111** [comment]
Update: This may have to do with the latest Firefox disabling add-ons for private browsing windows. It used to allow add-ons to run in private browsing windows by default, but now you have to enable the option. Testing YTNS now in a non-private window to see if it works as before. Will post update with results...

**Comment by jacekmusial** [comment]
This feature was first introduced in Firefox 67 on May 2019
https://www.mozilla.org/en-US/firefox/67.0/releasenotes/
https://support.mozilla.org/en-US/kb/extensions-private-browsing#w_extensions-in-private-windows

**Comment by ite-klass** [comment]
This is not an issue with the addon. It is a concept and feature of Firefox itself (not automatically allowing any addon to run in private mode).

There is nothing the addon could do or change to change this.

**Comment by jacekmusial** [comment]
Indeed, it's how the Firefox browser works. As it is not related to the plugin code, we can close this issue @lawfx @Ghostium
